docs/int/quickstart/quickstart-exit.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 6
+description: Exit a validator
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Exit a validator
+
+:::caution
+Charon is in an early alpha state and is not ready to be run on mainnet.
+:::
+
+Exiting your validator keys can be useful in situations where you want to stop staking and withdraw your staked ETH.
+
+## Pre-requisites
+
+- A quorum of operators needs to run the same exit command for the exit to succeed.
+- If any of the charon nodes restart after the exit command is run, it will lose the partial exit and you will have to begin again.
+
+## Step 1. Set the `EXIT_EPOCH`
+
+Set the `EXIT_EPOCH` environment variable to the epoch number at which you want to exit. For example, if you want to exit at epoch 112260, enter the following command:
+    
+    export EXIT_EPOCH=112260
+
+## Step 2. Run the `voluntary-exit` command
+
+Run the following voluntarily exit the validator:
+
+<Tabs groupId="validator-clients">
+  <TabItem value="teku" label="Teku" default>
+    <pre>
+      <code>
+    {String.raw`docker exec -ti charon-distributed-validator-node-teku-1 /opt/teku/bin/teku voluntary-exit \
+    --beacon-node-api-endpoint="http://charon:3600/" \
+    --confirmation-enabled=false \
+    --validator-keys="/opt/charon/validator_keys:/opt/charon/validator_keys" \
+    --epoch=$\{EXIT_EPOCH:-112260}`}
+      </code>
+    </pre>
+  </TabItem>
+  <TabItem value="lighthouse" label="Lighthouse">
+    <pre>
+      <code>
+    {String.raw`docker exec -ti charon-distributed-validator-node-lighthouse-1 /opt/lighthouse/lighthouse validator exit \
+    --beacon-node "http://charon:3600/" \
+    --validator-dir "/opt/charon/validator_keys" \
+    --epoch $\{EXIT_EPOCH:-112260}`}
+      </code>
+    </pre>
+  </TabItem>
+  <TabItem value="nimbus" label="Nimbus">
+    <pre>
+      <code>
+    {String.raw`docker exec -ti charon-distributed-validator-node-nimbus-1 /opt/nimbus-eth2/build/nimbus_beacon_node validator_exit \
+    --rpc="http://charon:3600/" \
+    --validators-dir="/opt/charon/validator_keys" \
+    --exit-epoch=$\{EXIT_EPOCH:-112260}`}
+      </code>
+    </pre>
+  </TabItem>
+</Tabs>
+
+Once a threshold of exit signatures has been reached, charon will submit the exit transaction to the beacon chain.
```

Comment:
```suggestion
Once a threshold of exit signatures has been received by any single charon client, it will craft a valid exit message and will submit it to the beacon chain for inclusion.
```